src/components/Detail/Detail.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, CardContent, Typography } from '@material-ui/core'
+import './Detail.css'
+
+const Detail = ({ name, cost, frequency }) => (
+  <div className="detail">
+    <Typography className="title text-content" variant="h6" gutterBottom>
+      Details
+    </Typography>
+    <Card>
+      <CardContent>
+        <Typography className="text-content" gutterBottom>
+          {name}
+        </Typography>
+        <Typography className="text-content" gutterBottom>
+          Cost: <span className="text-content-number">${cost}</span>
+        </Typography>
+        <Typography className="text-content">
+          Frequency: <span className="text-content-number">{frequency}/s</span>
+        </Typography>
+      </CardContent>
+    </Card>
+  </div>
+)
+
+Detail.defaultProps = {
```

Comment:
We going to use prop types?

Reply:
I've just added those default props because the story of this component needs some data to show something.